coord/src/dist_aggregate.c

```diff
@@ -195,35 +199,43 @@ typedef struct {
 } RPNet;
 
 static int getNextReply(RPNet *nc) {
-  while (1) {
```

Comment:
So the loop was never really needed?

Reply:
Probably initially intended to, but in practice you always exit after a single iteration. This function is used inside another loop